### ISSUE TITLE
"endpoint measured by assay" statements

### ISSUE THREAD

**Issue by vedina** [enanomapper/ontologies#73]
Is it possible to include statements in the ontology `endpoint X is measured by assay Y` ?
If yes, I'll provide a list to include.
If it is already in, please point me to.

**Comment by fehrhart** [comment]
Yes, possible IF you include that some endpoints can be measured by
multiple assays and vice versa. Depends also on the detailedness of the
assay or endpoint. E.g. LDH assay measures LDH activity of the supernatant
- this is one unique endpoint. But it tells you also about cell viability
another endopoint, living cells in %,  but this can also be measured by ATP
quantification or MTT.

**Comment by vedina** [comment]
Yes, sure, one endpoint can be measured by multiple assays and vice versa.  Do we have some of this already in the ontology?

**Comment by vedina** [comment]
Please see https://bugzilla.enanomapper.net/show_bug.cgi?id=276

**Comment by vedina** [comment]
Would it be possible to introduce statements which endpoint is measured by which technique, based on the  `Table 1: Characterization techniques` at page 71 of  
[OECD GUIDANCE ON SAMPLE PREPARATION AND DOSIMETRY FOR THE SAFETY TESTING OF
MANUFACTURED NANOMATERIALS](http://www.oecd.org/officialdocuments/publicdisplaydocumentpdf/?cote=env/jm/mono(2012)40&doclanguage=en)

**Comment by vedina** [comment]
Table 4 of  [doi:10.3109/17435390.2014.991431](http://www.tandfonline.com/doi/full/10.3109/17435390.2014.991431) contains the following list of endpoints measured by assay(s).  Is it possible to incorporate this information in the ontology?

|Assay (SOP) |	Endpoint |
| ---| ----|
|WST1 (measured in supernatants)	|Cytotoxicity|
|Lactate dehydrogenase (LDH)	|Cytotoxicity|
|PI uptake by flow cytometry	|Cytotoxicity|
|Plating efficiency	|Cytotoxicity|
|Relative growth activity|	Cytotoxicity|
|Trypan blue exclusion|	Cytotoxicity|
|MTT |	Cytotoxicity|
|ELISA	|Pro-inflammatory response|
|RTqPCR	|Pro-inflammatory response|
|ROS detection by DHE using plate readers	|Oxidative stress|
|ROS detection by carboxy-fluorescein	|Oxidative stress|
|Thiols detection by bromobimane using plate readers	|Oxidative stress|
|ROS detection by HE using flow cytometry	|Oxidative stress|
|RTqPCR	|Oxidative stress|
|Side scatter of laser light in flow cytometry|	Uptake|
|Cell fluorescence quantified by flow cytometry|	Uptake|
|Transmission electron microscopy (TEM)	|Uptake|
|Comet assay	|Genotoxicity:Single and double strand breaks|
|Comet assay with repair enzyme	|Genotoxicity: base DNA damage|
|Cytokinesis-block micronucleus assay	|Genotoxicity/mutagenicity|
|Barrier permeability (NaFlu)	|Barrier integrity|
|Barrier transport of NP in Transwells	|Transport of NPs across barrier|
|Phagocytic activity and respiratory burst of leukocytes by flow cytometry	|Immunotoxicity|
|Natural killer cell activity by flow cytometry|	Immunotoxicity|
|Proliferative response of lymphocytes by 3H thymidine incorporation|	Immunotoxicity|

**Comment by JKChang2015** [comment]
Added 12 endpoints, according to the table above.  470c96a4c953c6064abf2a9ee91d9f8b300261d7

The work spreadsheet: https://docs.google.com/spreadsheets/d/1C67Mv1_JOOACWHFSBjnXVL5NVs0fDPySpG0mVe2FsUA/edit#gid=2053541995
